Assignments/Splitwise/Python/ProcessInput.py

```diff
@@ -0,0 +1,76 @@
+from user import user
+
+class ProcessInput:
+    def __init__(self):
+        self.users = dict()
+        self.owes = dict()
+
+    # process 'ADD' function
+    def processAdd(self,inputList : List[str]):
+        usersString = inputList[2:]
+        count = 1
+        for usr in usersString:
+            id = 'u'+str(count)
+            self.users[id] = user(id,usr)
+            count += 1
+    
+    #process 'EXPENSE' function
+    def processExpense(self, inputList: List[str]):
+        owedTo = inputList[1]
+        amount = int(inputList[2])
+        num = int(inputList[3])
+        owedBy = inputList[4:4+num]
+
+        # Update the transaction 
+        def updateAmount(owedTo, usr, amt) :
+            if owedTo == usr : return
+            if (owedTo,usr) in self.owes:
+                self.owes[(owedTo,usr)] += amt
+            elif (usr,owedTo) in self.owes:
+                self.owes[(usr,owedTo)] -= amt
+            else:
+                self.owes[(owedTo,usr)] = amt
+
+
+        if "EQUAL" in inputList:
```

Comment:
The current code is not maintainable. Can you make it more modular ?

Reply:
This is the file from the previous approach. I forgot to remove it. I'll remove it now.